hex/sd_api_v3/sdk122_connectivity.patch

```diff
@@ -163,32 +179,20 @@ index 0f597ba..2f6bfb4 100644
-index a339754..8fb049b 100644
---- a/components/serialization/common/ble_serialization.h
-+++ b/components/serialization/common/ble_serialization.h
-@@ -122,7 +122,7 @@ typedef enum
- 
- /** Maximum length of p_value in \ref ble_gattc_write_params_t. See Bluetooth 4.0 spec: 3.4.5.1 and
-  *  3.4.5.3. */
--#define BLE_GATTC_WRITE_P_VALUE_LEN_MAX    (GATT_MTU_SIZE_DEFAULT - 3)
-+#define BLE_GATTC_WRITE_P_VALUE_LEN_MAX    (247 - 3)
- 
- /** See Bluetooth 4.0 spec: 3.4.4.7. */
- #define BLE_GATTC_HANDLE_COUNT_LEN_MAX     ((GATT_MTU_SIZE_DEFAULT - 1) / 2)
+               <IncludePath>..\..\..\config;..\..\..\..\..\..\components;..\..\..\..\..\..\components\ble\ble_dtm;..\..\..\..\..\..\components\ble\common;..\..\..\..\..\..\components\boards;..\..\..\..\..\..\components\drivers_nrf\clock;..\..\..\..\..\..\components\drivers_nrf\common;..\..\..\..\..\..\components\drivers_nrf\delay;..\..\..\..\..\..\components\drivers_nrf\hal;..\..\..\..\..\..\components\drivers_nrf\uart;..\..\..\..\..\..\components\libraries\bsp;..\..\..\..\..\..\components\libraries\crc16;..\..\..\..\..\..\components\libraries\log;..\..\..\..\..\..\components\libraries\log\src;..\..\..\..\..\..\components\libraries\queue;..\..\..\..\..\..\components\libraries\scheduler;..\..\..\..\..\..\components\libraries\timer;..\..\..\..\..\..\components\libraries\uart;..\..\..\..\..\..\components\libraries\util;..\..\..\..\..\..\components\serialization\common;..\..\..\..\..\..\components\serialization\common\struct_ser\s132;..\..\..\..\..\..\components\serialization\common\transport;..\..\..\..\..\..\components\serialization\common\transport\ser_phy;..\..\..\..\..\..\components\serialization\common\transport\ser_phy\config;..\..\..\..\..\..\components\serialization\connectivity;..\..\..\..\..\..\components\serialization\connectivity\codecs\common;..\..\..\..\..\..\components\serialization\connectivity\codecs\s132\middleware;..\..\..\..\..\..\components\serialization\connectivity\codecs\s132\serializers;..\..\..\..\..\..\components\serialization\connectivity\hal;..\..\..\..\..\..\components\softdevice\common\softdevice_handler;..\..\..\..\..\..\components\softdevice\s132\headers;..\..\..\..\..\..\components\softdevice\s132\headers\nrf52;..\..\..\..\..\..\components\toolchain;..\..\..\..\..\..\external\segger_rtt;..\config</IncludePath>
+diff --git a/components/serialization/common/ble_serialization.h b/components/serialization/common/ble_serialization.h    +
+index a339754..8fb049b 100644   
+--- a/components/serialization/common/ble_serialization.h   
++++ b/components/serialization/common/ble_serialization.h   
+@@ -122,7 +122,7 @@ typedef enum    
+    
+ /** Maximum length of p_value in \ref ble_gattc_write_params_t. See Bluetooth 4.0 spec: 3.4.5.1 and    
+  *  3.4.5.3. */    
+-#define BLE_GATTC_WRITE_P_VALUE_LEN_MAX    (GATT_MTU_SIZE_DEFAULT - 3)   
++#define BLE_GATTC_WRITE_P_VALUE_LEN_MAX    (247 - 3)   
```

Comment:
Is there a constant that could be used instead of `247` (and maybe also `3`)?

Reply:
There are no constants defined for it, but that could be added.